helpers/assignVar.js

```diff
@@ -11,9 +11,9 @@ const factory = globals => {
             throw new Error("assignVar helper key must be a string");
         }
 
-        // Validate that value is a string or integer
-        if (!utils.isString(value) && !Number.isInteger(value)) {
-            throw new Error("assignVar helper value must be a string or a number (integer)");
+        // Validate that value is a string or Number (int/float)
+        if (!utils.isString(value) && isNaN(value)) {
```

Comment:
I would use `Number.isFinite(value)` here for a more robust and less surprising implementation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite